v2/passwordless/pre-built-ui/setup/core/self-hosted-with-aws.mdx

```diff
@@ -0,0 +1,20 @@
+---  
+id: aws-setup-with-stacksnap  
+title: AWS Setup with Stacksnap  
+hide_title: true  
+---
+
```

Comment:
please replicate this across the following recipes:
- emailpassword
- thirdparty
- thirdparty + emailpassword
- thirdparty + passwordless
- session